packages/base/Tag/src/TagGroup/TagGroup.tsx

```diff
@@ -22,10 +14,15 @@ export const TagGroup = forwardRef<HTMLDivElement, Props>(function TagGroup(
 ) {
   const { children, className, ...rest } = props
 
-  const classes = useStyles()
-
   return (
-    <div {...rest} ref={ref} className={cx(classes.root, className)}>
+    <div
+      {...rest}
+      ref={ref}
+      className={twMerge(
+        'text-lg min-w-full transition-none -mr-2 -mb-2 [&>*]:mr-2 [&>*]:mb-2',
```

Comment:
Is there any difference between `mr-2` and `-mr-2`? Maybe it's a typo?
```suggestion
        'text-lg min-w-full transition-none mr-2 mb-2 [&>*]:mr-2 [&>*]:mb-2',
```

Reply:
@augustobmoura yeah, with `-` prefix we have negative value `margin-right: - 0.5rem`

Reply:
@OleksandrNechai noticed that Tailwind provides `space` util classes that we can maybe us to remove the `[&>*]`. Can you please check and try it out? https://tailwindcss.com/docs/space

Reply:
It can be done this way. `TagGroup` had this kind of styling with child elements before:
![image](https://github.com/toptal/picasso/assets/134127624/eb7cf891-cf3d-4515-9df2-15b67aa182e6)

So we need `[&>*]` here.